tests/test_000060.textproto

```diff
@@ -0,0 +1,150 @@
+# proto-file: third_party/libiamf/cli/proto/user_metadata.proto
+# proto-message: UserMetadata
+
+test_vector_metadata {
+  human_readable_description: "A stereo IAMF stream with two language labels."
+  file_name_prefix: "test_000060"
+  is_valid: true
+  mp4_fixed_timestamp: "2023-06-20 00:00:00"
+  temporal_units_per_ia_sample: 1
+  primary_tested_spec_sections: ["2.8/count_label", "2.8/language_label", "2.8/mix_presentation_annotations"]
+  base_test: "test_000005"
+}
+
+ia_sequence_header_metadata {
+  ia_code: 0x69616d66  # "iamf"
+  profile_name: PROFILE_VERSION_SIMPLE
+  profile_compatible: PROFILE_VERSION_SIMPLE
+}
+
+codec_config_metadata {
+  codec_config_id: 200
+  codec_config {
+    codec_id: 0x6970636d  # "ipcm"
+    num_samples_per_frame: 64
+    audio_roll_distance: 0
+    decoder_config_lpcm {
+      sample_format_flags: LPCM_LITTLE_ENDIAN
+      sample_size: 16
+      sample_rate: 16000
+    }
+  }
+}
+
+audio_element_metadata {
+  audio_element_id: 300
+  audio_element_type: AUDIO_ELEMENT_CHANNEL_BASED
+  reserved: 0
+  codec_config_id: 200
+  num_substreams: 1
+  audio_substream_ids: [0]
+  num_parameters: 0
+  scalable_channel_layout_config {
+    num_layers: 1
+    reserved: 0
+    channel_audio_layer_configs: [
+      {
+        loudspeaker_layout: 1  # Stereo
+        output_gain_is_present_flag: 0
+        recon_gain_is_present_flag: 0
+        reserved_a: 0
+        substream_count: 1
+        coupled_substream_count: 1
+      }
+    ]
+  }
+}
+
+mix_presentation_metadata {
+  mix_presentation_id: 42
+  count_label: 2
+  language_labels: ["en-us", "es-mx"]
+  mix_presentation_annotations_array: [
+    {
+      mix_presentation_friendly_label: "test_mix_pres"
+    },
+    {
+      mix_presentation_friendly_label: "prueba_mezcla_presentación"
+    }
+  ]
+  num_sub_mixes: 1
+  sub_mixes {
+    num_audio_elements: 1
+    audio_elements {
+      audio_element_id: 300
+      mix_presentation_element_annotations {
+        audio_element_friendly_label: "test_sub_mix_0_audio_element_0"
```

Comment:
count_label is 2, so the mix_presentation_element_annotations should have 2 audio_element_friendly_labels, refer [2.8. Mix Presentation OBU Syntax and Semantics](https://aomediacodec.github.io/iamf/#obu-mixpresentation)

Reply:
Resolved as of 31f5795.